301-sonarqube-azuresql/azuredeploy.json

```diff
@@ -0,0 +1,399 @@
+              "subnet": {
+                "id": "[variables('SQvmAppSubnetRef')]"
+              },
+              "publicIPAddress": {
+                "id": "[resourceId('Microsoft.Network/publicIPAddresses', variables('SQpublicIPName'))]"
+              }
+            }
+          }
+        ]
+      }
+    },
+    {
+      "name": "[parameters('sqVM_AppName')]",
+      "type": "Microsoft.Compute/virtualMachines",
+      "location": "[resourceGroup().location]",
+      "apiVersion": "2015-06-15",
+      "dependsOn": [
+        "[concat('Microsoft.Storage/storageAccounts/', variables('SQstorageName'))]",
+        "[concat('Microsoft.Network/networkInterfaces/', variables('SQvmAppNicName'))]"
+      ],
+      "tags": {
+        "displayName": "SQvmApp"
+      },
+      "properties": {
+        "hardwareProfile": {
+          "vmSize": "[variables('SQvmAppVmSize')]"
+        },
+        "osProfile": {
+          "computerName": "[parameters('sqVM_AppName')]",
+          "adminUsername": "[parameters('sqVM_AppAdmin_UserName')]",
+          "adminPassword": "[parameters('sqVM_AppAdmin_Password')]"
+        },
+        "storageProfile": {
+          "imageReference": {
+            "publisher": "[variables('SQvmAppImagePublisher')]",
+            "offer": "[variables('SQvmAppImageOffer')]",
+            "sku": "[variables('SQvmAppWindowsOSVersion')]",
+            "version": "latest"
+          },
+          "osDisk": {
+            "name": "SQvmAppOSDisk",
+            "vhd": {
+              "uri": "[concat('http://', variables('SQstorageName'), '.blob.core.windows.net/', variables('SQvmAppStorageAccountContainerName'), '/', variables('SQvmAppOSDiskName'), '.vhd')]"
+            },
+            "caching": "ReadWrite",
+            "createOption": "FromImage"
+          }
+        },
+        "networkProfile": {
+          "networkInterfaces": [
+            {
+              "id": "[resourceId('Microsoft.Network/networkInterfaces', variables('SQvmAppNicName'))]"
+            }
+          ]
+        }
+      },
+      "resources": [
+        {
+          "name": "configureAppVM_DSC",
+          "type": "extensions",
+          "location": "[resourceGroup().location]",
+          "apiVersion": "2015-06-15",
+          "dependsOn": [
+            "[concat('Microsoft.Compute/virtualMachines/', parameters('sqVM_AppName'))]"
+          ],
+          "properties": {
+            "publisher": "Microsoft.Powershell",
+            "type": "DSC",
+            "typeHandlerVersion": "2.9",
+            "autoUpgradeMinorVersion": true,
+            "settings": {
+              "modulesUrl": "[concat(variables('SQDSCFileLocation'), '/', variables('dscZipFile'))]",
+              "configurationFunction": "[variables('SQdscAppConfigurationFunction')]",
+              "properties": {
+                "nodeName": "[parameters('sqVM_AppName')]",
+                "connectionString": "[reference('deploySQLDB').outputs.jdbcConnString.value]",
```

Comment:
We do want to change this and the team is tracking this as an unresolved bug.  We haven't had a chance to remediate it yet, but the problem may fall into the same category as returning the Connection String in plain text in the Output of the database template.  We definitely want this updated for the secured release, but for this Beta, can we move forward with this as-is under the "this is an insecure template" caveat?